hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsSqsProperties.kt

```diff
@@ -24,13 +24,17 @@ data class HmppsSqsProperties(
     val visibilityTimeout: Int = 30,
     val errorVisibilityTimeout: Int = 0,
     val propagateTracing: Boolean = true,
+    val fifoQueue: String = "false",
```

Comment:
A String seems odd here - better as a Boolean?

Reply:
It's annoying - the API is inconsistent. You can supply "true" or "false" strings for fifoTopic but fifoQueue can only be "true" - no other value is supported.

If I make this a boolean I should really change fifoTopic as well.